### DISCUSSION TITLE
Set Schema default options for all schema?

### DISCUSSION THREAD

**Discussion by RodPaDev** [dynamoose/dynamoose#1349] [Q&A]
Is there a way of doing something similar to setting model defaults but for schemas?

```js
dynamoose.model.defaults.set({
  craete: false
})
```
something like this?
```js
dynamoose.schema.defaults.set({
  saveUnknown: false, 
  timestamps: true 
})
```

**Comment by fishcharlie** [comment]
Not currently. Submit an issue as a feature request for this if you can.

**Reply by RodPaDev** [reply]
Awesome will do so! And if it's simple enough I might make a PR too.

**Reply by fishcharlie** [reply]
That'd be awesome! It should be similar to the Model code. The only complication is that Model is a function whereas Schema is a class. I don't THINK that should make a major impact here tho.

**Reply by RodPaDev** [reply]
Should be the same thing, in the end JS classes are just sugarcoated functions
